include/Producer.hpp

```diff
@@ -41,6 +43,9 @@ namespace mediasoupclient
 		  webrtc::RtpSenderInterface* rtpSender,
 		  webrtc::MediaStreamTrackInterface* track,
 		  const nlohmann::json& rtpParameters,
+		  const bool stopTracks,
+		  const bool zeroRtpOnPause,
+		  const bool disableTrackOnPause,
```

Comment:
Bad rebase? There's quite a bit of previously-merged code here. Can we have it cleaned up before a final review?

Reply:
Possible. Moving the commits to the other branch was messy, tbh. It wouldn't allow a simple cherry-pick or rebase for some reason (the commits containing a merge commit, iirc).

Reply:
Better now?

Reply:
Apparently still the same! But it's reviewable overall, I think. :)

Reply:
I just rebased `3.3.0-daily` onto `3.4.0` and added two more commits on top for `5112`.